zeebe/auth/src/main/java/io/camunda/zeebe/auth/Authorization.java

```diff
@@ -14,4 +14,5 @@ public class Authorization {
   public static final String AUTHORIZED_USERNAME = "authorized_username";
   public static final String AUTHORIZED_CLIENT_ID = "authorized_client_id";
   public static final String USER_TOKEN_CLAIMS = "user_token_claims";
+  public static final String GROUPS_CLAIMS = "groups_claims";
```

Comment:
```suggestion
  public static final String USER_GROUPS_CLAIMS = "user_groups_claims";
```

❓ WDYT about renaming it to this, to indicate clearly these groups from the users IDP?